### DISCUSSION TITLE
[Feature] Non People Filter

### DISCUSSION THREAD

**Discussion by homeauto-git** [immich-app/immich#14399] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Hello,

I take a lot of landscape photos and often need to filter out images without any people. It would be incredibly helpful to have an additional filter in the "People" filter group for "No People."

I did search through the feature requests and couldn't find anything related to this. I'm also not sure if this functionality is already implemented—if it is, I apologize for missing it!

Thank you for considering this enhancement, and I appreciate all the work you’ve done on this fantastic software!

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
I'm pretty sure this is a duplicate request, but can't currently find the existing one.

**Reply by homeauto-git** [reply]
I looked again to make sure and I didn't fine a duplicate request. Did you find the one you are referencing?

**Reply by bo0tzz** [reply]
I did now, it would be covered by #10373
